src/text/Tag.scss

```diff
@@ -47,13 +47,13 @@
     color: var(--seeds-color-white);
   }
 
-  &[data-variant="in-process"] {
-    background-color: var(--seeds-color-highlight-light);
-    color: var(--seeds-color-highlight-darker);
+  &[data-variant="warn"] {
```

Comment:
Updated the naming here to warn - we don't have an `in-process` in the Figma tokens, and the colors are the same (confirmed with design)